data/conditions.ts

```diff
@@ -388,17 +388,7 @@ export const Conditions: {[k: string]: ConditionData} = {
 			target.removeVolatile('Protect');
 			target.removeVolatile('Endure');
 
-			if (data.source.hasAbility('infiltrator') && this.gen >= 6) {
-				data.moveData.infiltrates = true;
-			}
-			if (data.source.hasAbility('normalize') && this.gen >= 6) {
-				data.moveData.type = 'Normal';
-			}
```

Comment:
Why this change?

Reply:
I removed them all thinking they were all redundant, but apparently the `onModifyMove` and `onModifyType` events don't run which is why they were there in the first place (and why Adaptability doesn't need to be there anymore since that's on a new event). I added them back now.